src/server/lib/spreadsheet.js

```diff
@@ -172,6 +175,21 @@ async function spreadsheetToDocuments (
           // omit these sheets
           return
 
+        case 'cover':
+          // expenditure category is only specified in cover sheet, save it here to
+          // populate in individual project "documents"
+          detailedExpenditureCategory = jsonRow['detailed expenditure category']
+          break
+
+        case 'ec 1 - health':
+        case 'ec 2 - economic impact':
+        case 'ec 3 - services':
+        case 'ec 4 - premium pay':
+        case 'ec 5 - infrastructure':
+        case 'ec 7 - admin':
+          jsonRow['detailed expenditure category'] = detailedExpenditureCategory
+          break
```

Comment:
This is definitely weird.  Let me know if you can think of a cleaner approach to associate detailed expenditure category with individual project "documents".